book.toml

```diff
@@ -21,8 +21,10 @@ git-branch = "main"
 
 [output.html.redirect]
 "/datasets/mozetl/search_aggregates/reference.html" = "/datasets/search/search_aggregates/reference.html"
-"/datasets/mozetl/search_clients_daily/reference.html" = "/datasets/search/search_clients_daily/reference.html"
-"/datasets/mozetl/search_clients_last_seen/reference.html" = "/datasets/search/search_clients_last_seen/reference.html"
+"/datasets/mozetl/search_clients_daily/reference.html" = "/datasets/search/search_clients_engines_sources_daily/reference.html"
+"/datasets/search/search_clients_daily/reference.html" = "/datasets/search/search_clients_engines_sources_daily/reference.html"
+"/datasets/mozetl/search_clients_last_seen/reference.html" = "/datasets/search/search_clients_engines_sources_last_seen/reference.html"
```

Comment:
@fbertsch points out that the last_seen tables actually _are_ aggregated to the client level rather than (client, engine, source), so we shouldn't be doing this renaming!

I will file a bigquery-etl PR to back out the new last_seen views, and then update this PR accordingly.

Reply:
See https://github.com/mozilla/bigquery-etl/pull/2050

Reply:
Oh thanks for verifying

Reply:
Thanks for checking

Reply:
I've now updated this PR to exclude the erroneous last_seen changes.